### ISSUE TITLE
BT-01 (Procedure Legal Basis); National legal bases; [CODELIST]

### ISSUE THREAD

**Issue by tinne** [eForms/eForms#258]
From #86, I learn that with a national notice I specify the Procedure Type with BT-105, if it fits, and with BT-88 else (where BT-03 Notice Type can most probably be used for further differentiation, if necessary).

Following up #57, now there is a code point “Other” in the Procedure Legal Basis list, is there a preferred Business Term where to put the national legal basis at hand - BT-88 or elsewhere?

**Comment by JachymHercher** [comment]
No, there is not. The BT-88 is intended about the practical information about the procedure rather than the legal basis. We assumed that submitting the legal basis would be considered as administrative burden and it wouldn't be considered too interesting for most users. 

If you say this would be interesting (an example use case or two would be appreciated), then there are essentially two possible approaches. One would be to have an additional BT to store the more detailed legal basis; the second would be to use BT-01 and expand the codelist. 

We would be inclined to suggest the latter option, i.e. instead of "other" there could be "other (national law X)", "other (national law Y)", etc. Alternatively, any [European Legislation Identifier](https://eur-lex.europa.eu/eli-register/about.html) could perhaps be used.

(FYI @muricna @yvesjo)

**Comment by tinne** [comment]
My main concern it the huge current differences between German below-thresholds notices regarding the jurisdictions for construction vs. general / non-construction, see my last year's draft mapping of eForms to VOB (construction) and UVgO (mainly supplies and services) https://github.com/eForms/eForms/issues/169#issuecomment-424779939 which I suspect will not be fully integrated with each other by 2023, but also special tendering law such as German Budget Law, maybe the national equivalent to these EU institution procuments.

I am not whether a centrally fostered code list would be appropriate. The ELI link to MDR in the [Governance rules document](https://eur-lex.europa.eu/content/eli-register/governance_rules.pdf) seems to be currently broken, so I do not yet understand whether this could help.

**Comment by JachymHercher** [comment]
(@tinne the link seems to be working again, not sure if that helps)

**Comment by tinne** [comment]
@JachymHercher After consultation with the people working on [electronic legislation “eGesetzgebung”](http://egesetzgebung.bund.de/index_en.html) there seems to be some trouble with ELI, most notably the lack of it to support draft laws.

With [LegalDocML](https://www.oasis-open.org/committees/legaldocml) a competing approach to a common legal document standard appears to be the most promising approach to national law identification.

Would allowing an LegalDocML **bill** identifier be an approach you could be comfortable with?

**Comment by JachymHercher** [comment]
@tinne We've also discussed internally and having just a single identifier - ELI - seems easier to us. In particular, in the context of procurement at least, a lack of support for draft legislation shouldn't cause any problems. Furthermore, apparently, it should be possible to use ELI together with LegalDocML, with the former being an identifier and the latter primarily an XML schema. 

Would you have any more information about the "LegalDocML bill identifier"? 

In any case, translating another identifier into LEI shouldn't pose a major challenge, no? 

(For reference https://eur-lex.europa.eu/eli-register/implementation.html gives information on the state of implementation of the LEI in the EU.)

**Comment by jpmckinney** [comment]
The current forms use CELEX numbers. Is there a broad transition to ELI from CELEX in the EC/EU?

**Comment by tinne** [comment]
@JachymHercher 

> Would you have any more information about the "LegalDocML bill identifier"?

This should be available in the links I quoted above - LegalDocumentML is an OASIS standard available at https://www.oasis-open.org/committees/legaldocml. Bills are “deliberative documents produced by parliamentary activities or from other empowered bodies” (cf. chap. 4.2.1 in http://docs.oasis-open.org/legaldocml/akn-core/v1.0/akn-core-v1.0-part1-vocabulary.html) and thus form a very general single identifier to any kind of act of legislation. 

> In any case, translating another identifier into LEI shouldn't pose a major challenge, no?

The ELI documentation seems a bit fuzzy, but from the change log I read “It is now possible to use the URI ‘application/akn+xml’ as a value of eli:format to indicate an AkomaNtoso format”, thus I suspect ELI as of version 1.2 might allow to superset LegalDocumentML entities. (It also contains a declaration of a “local ID” which seems to be a backward compatibility mechanism to the CELEX ID system.)

**Comment by JachymHercher** [comment]
Thank you. I'm still a bit lost in the Akoma Ntoso URIs, but I just haven't done enough reading (incl. section 5.9.1 Identification). 

If ELI is compatible with LegalDocumentML and CELEX, then we don't have a problem, right? In any case, this probably needs a domain expert - @muricna / colleague of hers.
